ci-operator/step-registry/api-job/api-job-commands.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+prow_api_token=$(cat "/var/run/vault/tests-private-account/prow-api-token")
+export APITOKEN=${prow_api_token}
+
+github_token=$(cat "/var/run/vault/tests-private-account/token-git")
+export GITHUB_TOKEN=${github_token}
+
+job get_payloads 4.11.0,4.12.0,4.13.0 --push true --run true
```

Comment:
@jianzhangbjz better change to `python3 job get_payloads 4.11.0,4.12.0,4.13.0 --push true --run true`

Reply:
No need, since `job` installed as a command.